test/sparse/sparse.jl

```diff
@@ -1779,6 +1779,21 @@ end
     show(ioc, MIME"text/plain"(), sparse(Int64[1, 1], Int64[1, 2], [1.0, 2.0]))
     @test String(take!(io)) == "1×2 SparseMatrixCSC{Float64,Int64} with 2 stored entries:\n  ⋮"
 
+    # even number of rows
+    ioc = IOContext(IOContext(io, :displaysize => (8, 80)), :limit => true)
+    show(ioc, MIME"text/plain"(), sparse(Int64[1,2,3,4], Int64[1,1,2,2], [1.0,2.0,3.0,4.0]))
```

Comment:
(Another tangent: That methods of `show` accepting a stream aren't `!`-terminated seems inconsistent with the usual convention?)